tests/acceptance/live-test.js

```diff
@@ -0,0 +1,14 @@
+import { module, test } from 'qunit';
+import { visit, currentURL } from '@ember/test-helpers';
+import { setupApplicationTest } from 'website-www/tests/helpers';
+
+module('Acceptance | live', function (hooks) {
+  setupApplicationTest(hooks);
+
+  test('visiting /live', async function (assert) {
+    await visit('/live');
+
+    assert.strictEqual(currentURL(), '/live');
+    assert.dom('.btn-share').hasText('Screenshare');
```

Comment:
Ember provides us data-test selectors specifically for tests so I think we can use that here.

Reply:
Okay, I can see we are using data-test everywhere in tests. I thought we use test id as the least priority selector in testing from [react-testing-library](https://testing-library.com/docs/queries/about/#priority). Anyways, thank you I will make the change.